src/api/ApiAccessTokenProvider.tsx

```diff
@@ -9,15 +9,9 @@ interface Props {
 }
 
 export const ApiAccessTokenProvider: FC<Props> = ({ children }) => {
-  const [isReady, loading] = useIsAuthorizationReady();
+  const [isReady, loading, , gotTokensOnce] = useIsAuthorizationReady();
```

Comment:
Unnecessary double-comma here.

Reply:
Not actually. The hook returns array of 4 items and the 3rd is useless here, but needs to be defined with an empty slot.

There would be an error if an (unused) variable is set here, so left it empty.

Reply:
Oh, right, then it's ok.